ocf_data_sampler/load/load_dataset.py

```diff
@@ -0,0 +1,55 @@
+""" Loads all data sources """
+import xarray as xr
+
+from ocf_data_sampler.config import Configuration
+from ocf_data_sampler.load.gsp import open_gsp
+from ocf_data_sampler.load.nwp import open_nwp
+from ocf_data_sampler.load.satellite import open_sat_data
+from ocf_data_sampler.load.site import open_site
+
+
+def get_dataset_dict(config: Configuration) -> dict[xr.DataArray, dict[xr.DataArray]]:
+    """Construct dictionary of all of the input data sources
+
+    Args:
+        config: Configuration file
+    """
+
+    in_config = config.input_data
+
+    datasets_dict = {}
+
+    # Load GSP data unless the path is None
+    if in_config.gsp and in_config.gsp.gsp_zarr_path:
+        da_gsp = open_gsp(zarr_path=in_config.gsp.gsp_zarr_path).compute()
+
```

Comment:
Maybe I'm missing something, but why does it need a `compute()` here?

Reply:
It was added in this PR https://github.com/openclimatefix/ocf-data-sampler/pull/49, some reasoning is given in the PR description, I imagine it won't make much of a difference either way

Reply:
Ah thanks! Missed that

Reply:
So, ill leave this?

Reply:
Yeah! My bad